fix(cloudflare): error if cacheDir already exists (#168)
withastro/adapters: .changeset/eleven-books-push.md

---
"@astrojs/cloudflare": patch
---

> Fixes an error which occured if the cache directory already exist, causing `astro dev` to fail

Reviewer: ```suggestion
Fixes an error where the `astro dev` command would fail if the cache directory already existed
```

How about this? Is this correct?

If you don't like it, the original is fine if you change `already exist` to `already existed`
